webauthn/registration/verify_registration_response.py

```diff
@@ -138,7 +142,7 @@ def verify_registration_response(
                 f'Unexpected token_binding status of "{status}", expected one of "{",".join(expected_token_binding_statuses)}"'
             )
 
-    attestation_object = parse_attestation_object(response.attestation_object)
+    attestation_object = parse_attestation_object(response.attestation_object)  # would be nice to enclose this one in try - except ValidationError and raise InvalidRegistrationResponse
```

Comment:
Can you create a new issue about this and make this comment a `# TODO: Issue #NNN` to point to the issue? It's a worthwhile improvement suggestion, but maybe not for this PR.

Reply:
Created